packages/utils-test/src/setupFixture.ts

```diff
@@ -23,10 +26,15 @@ export async function setupFixture(fixtureName: string) {
   }
 
   const tempDir = tempy.directory();
-  const cwd = path.join(tempDir, fixtureName);
+  const tmpIndexSegment = `${tmpIndex++}`;
+  const cwd = path.join(tempDir, tmpIndexSegment, fixtureName);
```

Comment:
Why do we need `tmpIndexSegment`?

Reply:
I found that note that I added the git init in creating the fixtures, I can’t initialize again. What I’m doing is isolating fixture instances per test setup fixture call.